testutil/environment/clean.go

```diff
@@ -1,4 +1,4 @@
-package environment // import "github.com/docker/docker/internal/test/environment"
+package environment
```

Comment:
why was this removed?

Reply:
Only one of these is needed per package; since it doesn't do anything having them in multiple files and I was having to touch the line anyways I figured I'd remove them so that it wouldn't be a part of the diff in the future if this package is ever moved again.

Reply:
Removing them adds more code-churn (trying to avoid that); We should probably have a validate step to make sure packages have these comments; would it make it easier  to have them in all files (even if redundant)?

Reply:
Although note that I didn't audit every package to see if it needed the extra import lines removed, I just happened to notice it in one or two and figured incremental cleanup would be good :)

Reply:
> Removing them adds more code-churn (trying to avoid that)

Fair enough, although note that I had to touch these lines anyways to update them, so I just thought if we were updating them we might as well remove them and save code churn next time (if there is a next time, hopefully not).

> We should probably have a validate step to make sure packages have these comments; would it make it easier to have them in all files (even if redundant)?

I don't think so, it just seems more likely that a typo could break things because we end up with conflicting import comments to me, but I don't think it's super important either way.

Let me know if you'd like me to add them back here, I still need to figure out why CI is failing so I can do this before pushing the next update.

Reply:
Ping @thaJeztah on this last question; would you prefer that I revert this change? Thanks!

Reply:
oh! sorry, missed your last comment.

Yes, I'd prefer reverting it.

Reply:
actually; let's work on a test to make sure every package has the comment set. I wonder if that also would help with some packages still showing up in the wrong location on godoc.org https://godoc.org/github.com/moby/moby

Reply:
As long as it's set once it should show up in the right place on godoc, it doesn't need to be set in every file though.